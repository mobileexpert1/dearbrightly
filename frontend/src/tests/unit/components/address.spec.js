import React from 'react';
import { shallow } from 'enzyme';

import Address from '../../../sections/dashboard/pages/Address';

describe('Address', () => {
    it('should render correctly', () => {
        const component = shallow(<Address />);
        expect(component).toMatchSnapshot();
    });
});
