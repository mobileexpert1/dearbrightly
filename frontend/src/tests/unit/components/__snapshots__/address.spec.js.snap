// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Address should render correctly 1`] = `
<div
  className="address-container"
>
  <div>
    Street
  </div>
  <div>
    City, State, Zip
  </div>
  <div>
    Phone
  </div>
</div>
`;
